src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -419,6 +422,7 @@ private void updateRetentionTime(final EventType original, final EventType event
             } else {
                 eventType.setOptions(original.getOptions());
             }
+            timelineService.repartitionET(eventType, getNumberOfPartitionsFromStats(eventType.getDefaultStatistic()));
```

Comment:
what if you fo not need to repartition event type at this point? partitions number did not change